### PR TITLE
Documentation update for changes to Arcade Validation

### PR DESCRIPTION
**Pull request by missymessa**
### To double check:

* [x] The right tests are in and and the right validation has happened.  Guidance:  https://github.com/dotnet/core-eng/tree/master/Documentation/Validation

Related to: https://github.com/dotnet/core-eng/issues/11447

**NOTE**: While this PR may be approved, I won't merge it in until the rest of the automation work has been completed in the aforementioned issue so that the documentation and the actual functionality is consistent.

### CODE REVIEW

File: Documentation/Validation/Overview.md [hunk 1]
```diff
@@ -23,10 +22,18 @@ Exceptions:
 
 You can try this [locally](https://github.com/dotnet/arcade/blob/master/eng/validate-sdk.cmd).
 
+To validate against the Arcade Validation for Promotion pipeline (that includes the ability to build Arcade with the bellwether repos), follow these steps (which are similar to the steps outlined here for [How to Validate a Private Build](https://github.com/dotnet/arcade/blob/master/Documentation/Policy/TestingMSBuildGuidance.md#how-to-validate-a-private-build)):
+
+1. Run a build of your Arcade branch on the [arcade-official-ci](https://dnceng.visualstudio.com/internal/_build?definitionId=6) Azure DevOps Pipeline
+2. [Promote your build](../Darc.md#add-build-to-channel) to the "General Testing" Maestro channel. 
+3. Create a branch of [Arcade Validation](https://github.com/dotnet/arcade-validation)
```
Comment (riarenas) [inline Documentation/Validation/Overview.md:29]: [nit]
image


```suggestion
3. Create a branch of [arcade-validation](https://github.com/dotnet/arcade-validation)
```

File: Documentation/Validation/Overview.md [hunk 2]
```diff
@@ -71,8 +72,6 @@ Details on how sending jobs to Helix works [here](https://github.com/dotnet/arca
 
 **Publishing to BAR**
 
-When the Build portion of the validation build completes we publish the produced package to BAR. This
```
Comment (riarenas) [inline Documentation/Validation/Overview.md:74]: What's the change here? just the formatting?

File: Documentation/Validation/Overview.md [hunk 3]
```diff
@@ -23,10 +22,18 @@ Exceptions:
...
+
+1. Run a build of your Arcade branch on the [arcade-official-ci](https://dnceng.visualstudio.com/internal/_build?definitionId=6) Azure DevOps Pipeline
+2. [Promote your build](../Darc.md#add-build-to-channel) to the "General Testing" Maestro channel. 
+3. Create a branch of [Arcade Validation](https://github.com/dotnet/arcade-validation)
+4. Using darc, run `update-dependencies` ([update-dependencies documentation](../Darc.md#updating-dependencies-in-your-local-repository)) on your Arcade Validation branch to use the build of Arcade you just created in the previous steps. 
+5. Push your branch up to Azure DevOps Arcade Validation repository and run a build of your branch on the [dotnet-arcade-validation-for-promotion](https://dev.azure.com/dnceng/internal/_build?definitionId=838&_a=summary) to verify your changes. 
+6. It's not necessary to merge your Arcade Validation branch into the repo's main branch, so feel free to delete it when you're done validating your changes.
+
 ### '.NET Tools - Validation' channel
```
Comment (mmitche) [inline Documentation/Validation/Overview.md:34]: I think this (and the line below) can be removed. The "before" time is so old now that I don't think it's important for context any longer.

**Review by mmitche — APPROVED** [review]

File: Documentation/Validation/Overview.md [hunk 4]
```diff
@@ -1,17 +1,16 @@
 # Arcade Validation
 
-We need to make sure changes done in the Arcade SDK as well as in the [core packages](https://github.com/dotnet/arcade/tree/master/Documentation/CorePackages) 
-don't break any of the consuming repos or Arcade itself. 
+We need to make sure changes done in the Arcade SDK as well as in the [core packages](https://github.com/dotnet/arcade/tree/master/Documentation/CorePackages) don't break any of the consuming repos or Arcade itself. 
 
 ## Arcade Validation Policy
 
-With the goal of being more transparent, eliminating surprises, and minimizing disruptions, we now will **only** deploy Arcade and/or machine (image) updates when the following criteria is met:
 - Each bell-weather repo (defined as runtime, aspnetcore, installer) must be building green so build problems won’t be compounded by new Arcade versions or images.
```
[the remark this answers is not in the document]
Comment (missymessa) [reply Documentation/Validation/Overview.md:7]: whoops!!

File: Documentation/Validation/Overview.md (hunk 2 above)
Comment (missymessa) [reply Documentation/Validation/Overview.md:74]: Yeah, there were a bunch of manual line breaks and it bugged me, so I changed them at the risk of having more edits in my PR than necessary 😊

File: Documentation/Validation/Overview.md (hunk 3 above)
Comment (missymessa) [reply Documentation/Validation/Overview.md:34]: Thanks for the permission to remove this 😄 I didn't think it was relevant anymore and wanted to remove it, too :)

File: Documentation/Validation/Overview.md [hunk 5]
```diff
@@ -1,17 +1,16 @@
...
-- Arcade updates has been tested with each bell-weather repo.
-
-Exceptions:
-- Servicing continues as is today, with possible conflicts arising with images which we’ll need to deal on a case by case basis.  (this should also go away w/ future plans)
-- If an Arcade or machine queue update (includes VS) is needed to unblock any build, then obviously we’ll do that.  (but with targeted changes)
+- Contributors who are changing existing code should use their best judgement to decide if additional validation against the "bellwether repos" is necessary. The following is a list of situations in which the contributor may want to run their changes against the bellwether repos:
```
[the remark this answers is not in the document]
Comment (missymessa) [reply Documentation/Validation/Overview.md:8]: That's a good point, too. I do notice folks doing that in Arcade changes that may only affect certain builds/pipelines, but it's a good thing to have written down as a reminder.

**Pull request merged**